scripts/build/Dockerfile.riscv64-stable-cross.tmpl

```diff
@@ -17,29 +17,18 @@ RUN dpkg --add-architecture ${DEBIAN_ARCH} && \
 # Install required packages
 RUN apt-get install -y --no-install-recommends \
 	build-essential \
-	pkg-config \
-	git \
 	crossbuild-essential-${DEBIAN_ARCH}	\
-	libc6-dev-${DEBIAN_ARCH}-cross		\
-	libc6-${DEBIAN_ARCH}-cross		\
+	git 					\
 	libbz2-dev:${DEBIAN_ARCH}		\
+	libc6-${DEBIAN_ARCH}-cross		\
+	libc6-dev-${DEBIAN_ARCH}-cross		\
 	libexpat1-dev:${DEBIAN_ARCH}		\
-	ncurses-dev:${DEBIAN_ARCH}		\
 	libssl-dev:${DEBIAN_ARCH}		\
+	ncurses-dev:${DEBIAN_ARCH}		\
+	pkg-config				\
 	protobuf-c-compiler			\
 	protobuf-compiler			\
 	python3-protobuf			\
-	libnl-3-dev:${DEBIAN_ARCH}		\
-	libprotobuf-dev:${DEBIAN_ARCH}		\
-	libnet-dev:${DEBIAN_ARCH}		\
-	libprotobuf-c-dev:${DEBIAN_ARCH}	\
-	libcap-dev:${DEBIAN_ARCH}		\
-	libaio-dev:${DEBIAN_ARCH}		\
-	uuid-dev:${DEBIAN_ARCH}			\
-	libnl-route-3-dev:${DEBIAN_ARCH}	\
-	libnftables-dev:${DEBIAN_ARCH}	\
-	libgnutls28-dev:${DEBIAN_ARCH}	\
-	iproute2:${DEBIAN_ARCH}	\
 	$(awk '!/^#/ && NF {print $0 ":'"${DEBIAN_ARCH}"'"}' /tmp/apt-packages.txt) \
```

Comment:
apt-packages.txt contains protobuf-c-compiler, doesn't it? Do we really need to install protobuf-c-compiler:${DEBIAN_ARCH} package?